.github/workflows/gradle.yml

```diff
@@ -0,0 +1,26 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Java CI with Gradle
+
+on:
+  push:
+    branches: [ master, develop, feature/code-cleanup ]
```

Comment:
Do we need for `feature/code-cleanup`?

Reply:
No, I forgot to remove it (I was working with friends and needed to use it on this branch on my personal repo).
I'll remove it 👍